app/src/main/AndroidManifest.xml

```diff
@@ -96,7 +96,8 @@
     <activity android:name=".app.settings.profile.ProfileEditActivity" />
     <activity
       android:name=".app.settings.profile.ProfileListActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/profile_list_activity_title"/>
```

Comment:
Add space before `/>`.

Press `Control + Alt + L` - reformatting shortcut to fix the formatting.